### PR TITLE
Add bcc option

### PR DESCRIPTION
**Pull request by davydovanton**
Reference to #56

/cc @hanami/contributors and @hanami/core-team

### CODE REVIEW

File: lib/hanami/mailer/dsl.rb [hunk 1]
```diff
@@ -229,6 +230,95 @@ def cc(value = nil)
...
+      # Sets the bcc (blind carbon copy) for mail messages
+      #
+      # It accepts a hardcoded value as a string or array of strings.
+      # For dynamic values, you can specify a symbol that represents an instance
+      # method.
+      #
+      # This value is optional.
+      #
+      # When a value is given, specify the bcc of the email. Otherwise,
+      # it returns the bcc of the email
```
Comment (cllns) [inline lib/hanami/mailer/dsl.rb]: What about:
```
# When a value is given, it specifies the bcc for the email. 
# When a value is not given, it returns the bcc of the email.
```
? That is a little more clear to me

Comment (davydovanton) [inline lib/hanami/mailer/dsl.rb]: hmm. yes, you solution looks more clear 👍

File: lib/hanami/mailer/dsl.rb [hunk 2]
```diff
@@ -229,6 +230,95 @@ def cc(value = nil)
...
+      # @overload bcc(value)
+      #   Sets the bcc
+      #   @param value [String, Array, Symbol] the hardcoded value or method name
+      #   @return [NilClass]
+      #
+      # @overload bcc
+      #   Returns the bcc
+      #   @return [String, Array, Symbol] the recipient
+      #
+      # @since 0.1.0
```
Comment (joneslee85) [inline lib/hanami/mailer/dsl.rb]: is it?

**Comment by joneslee85** [comment]
@davydovanton looks +1

File: lib/hanami/mailer/dsl.rb (hunk 2 above)
Comment (davydovanton) [inline lib/hanami/mailer/dsl.rb]: nice catch 👍 fixed (and for `cc` too)

File: lib/hanami/mailer/dsl.rb [hunk 3]
```diff
@@ -229,6 +230,95 @@ def cc(value = nil)
...
+      #
+      #     private
+      #
+      #     def recipients
+      #       users.map(&:email)
+      #     end
+      #   end
+      #
+      #   other_users = [User.new(name: 'L'), User.new(name: 'MG')]
+      #   WelcomeMailer.deliver(users: users)
```
Comment (artofhuman) [inline lib/hanami/mailer/dsl.rb]: Should this be `other_users` instead `users` ?

**Comment by jodosha** [comment]
@davydovanton Thanks for this PR. Merging it!

**Comment by davydovanton** [comment]
🎉

**Pull request merged**